### PR TITLE
Chrome Manifest v3

### PR DESCRIPTION
**Pull request by gabrielfalcao**
Here goes nothing

### CODE REVIEW

File: Safari/grammarannoyer.safariextension/grammar.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ function handleText(textNode)
                 "whom": new RegExp(/\bwho\b/ig),
                 "to": new RegExp(/\btoo\b/ig),
                 "than": new RegExp(/\bthen\b/ig),
-                "leopard": new RegExp(/\bkeyboard\b/ig)
```
Comment (sursh) [inline Safari/grammarannoyer.safariextension/grammar.js:39]: Ha!

File: Chrome/manifest.json [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-	"version": "1.0",
+	"version": "1.0.1",
 	"description": "Annoy you're friends! Makes common grammar mistakes across the web.",
-	"content_scripts": 
+	"content_scripts":
 	[
 		{
 			"matches": ["*://*/*"],
-			"js": ["content_script.js"],
-			"run_at": "document_end"
```
Comment (sursh) [inline Chrome/manifest.json:11]: Curious, as I am not a web dev, why change this?

**Review by sursh — APPROVED** [review]
What a blast from the past! How on earth did you find this? Have you actually used it as a prank?

File: Chrome/manifest.json (hunk 2 above)
Comment (gabrielfalcao) [reply Chrome/manifest.json:11]: It appears to be merely an upgrade to the manifest v3 interface 😬

**Comment by gabrielfalcao** [comment]
> What a blast from the past! How on earth did you find this? Have you actually used it as a prank?

I don't totally remember but I think I was looking into Recurse Center's projects and somehow landed at your's 😶‍🌫️

**Comment by gabrielfalcao** [comment]
> Have you actually used it as a prank?
No aminals were hurt in the making of this

**Pull request merged**